yesod-auth/Yesod/Auth/Email.hs

```diff
@@ -174,6 +175,18 @@ data ForgotPasswordForm = ForgotPasswordForm { _forgotEmail :: Text }
 data PasswordForm = PasswordForm { _passwordCurrent :: Text, _passwordNew :: Text, _passwordConfirm :: Text }
 data UserForm = UserForm { _userFormEmail :: Text }
 data UserLoginForm = UserLoginForm { _loginEmail :: Text, _loginPassword :: Text }
+data VerifyEmailMode = AlwaysSendVerify | OnlySendOnUnVerified
+
+defaultVerifyEmailMode :: VerifyEmailMode
+defaultVerifyEmailMode = alwaysSendVerify
+
+alwaysSendVerify :: VerifyEmailMode
```

Comment:
Since it's the default value, I'm not convinced it should be. I was thinking we'd only export the values that would be used to override default implementation.